### DISCUSSION TITLE
Hide all tools?

### DISCUSSION THREAD

**Discussion by Zacharia-Schmitz** [lunapaint/vscode-luna-paint#191] [Q&A]
Type: <b>Feature Request</b>

Using Luna Paint as an image viewer and both editor. If I want to just view the image file, I am wondering if there is a way to hide all tools with a hotkey.

Extension version: 0.16.0
VS Code version: Code 1.81.0 (6445d93c81ebe42c4cbd7a60712e0b17d9463e97, 2023-08-02T12:38:28.722Z)
OS version: Darwin arm64 22.6.0
Modes:

**Comment by Tyriar** [comment]
Yes you can hide all windows with F5, F6, F7 and F8. This is inspired by Paint.NET's shortcuts to do the same thing.
